backend/typescript/seeders/seed.ts

```diff
@@ -0,0 +1,186 @@
+    ),
+    status: "Approved",
+    volunteer_needed: true,
+    volunteer_id: 1,
+    volunteer_time: "8:00 AM",
+    frequency: "Monthly",
+    recurring_donation_id: "2",
+    recurring_donation_end_date: new Date("2021-10-01T00:06:00.000Z"),
+    notes: "these are the copied notes",
+    createdAt: new Date(1636226732806),
+    updatedAt: new Date(1636226732806),
+  },
+];
+
+const seedContent = [
+  {
+    food_rescue_description:
+      "Food rescue shifts assist with donations, either through unloading at the fridge or picking up off-site and bringing it to the fridge. For more information, check out the link here:",
+    food_rescue_url: "https://www.google.com/",
+    checkin_description:
+      "Community fridge check-in shifts are done daily to maintain health standards. Tasks include inventory and cleaning. Note: shifts typically take half an hour, and you can arrive any time within your time window. For detailed instructions, click this link:",
+    checkin_url: "https://www.google.com/",
+    createdAt: new Date(1636226732806),
+    updatedAt: new Date(1636226732806),
+  },
+];
+
+const seedCheckins = [
+  {
+    volunteer_id: 1,
+    start_date: new Date("1 March 2022"),
+    end_date: new Date("2 May 2022"),
+    notes: "Please remember to bring your own handsanitizer!",
+    createdAt: new Date(1647818578000),
+    updatedAt: new Date(1649989378000),
+  },
+  {
+    start_date: new Date(),
+    end_date: new Date(),
+    notes:
+      "Make sure to follow proper health and safety guidelines before coming to the fridge.",
+    is_admin: true,
+    createdAt: new Date(1647818578000),
+    updatedAt: new Date(1649989378000),
+  },
+  {
+    volunteer_id: 1,
+    start_date: new Date(),
+    end_date: new Date(),
+    notes: "Please remember to bring your own handsanitizer!",
+    createdAt: new Date(1647818578000),
+    updatedAt: new Date(1649989378000),
+  },
+];
+
+export const up: Seeder = async ({ context: sequelize }) => {
+  await sequelize.getQueryInterface().bulkInsert("users", seedUsers);
+  await sequelize.getQueryInterface().bulkInsert("donors", seedDonors);
+  await sequelize.getQueryInterface().bulkInsert("volunteers", seedVolunteers);
+  await sequelize.getQueryInterface().bulkInsert("scheduling", seedSchedules);
+  await sequelize.getQueryInterface().bulkInsert("content", seedContent);
+  await sequelize.getQueryInterface().bulkInsert("checkin", seedCheckins);
+};
+export const down: Seeder = async ({ context: sequelize }) => {
+  await sequelize
+    .getQueryInterface()
+    .dropTable("scheduling", { cascade: true });
+  await sequelize.getQueryInterface().dropTable("donors", { cascade: true });
+  await sequelize.getQueryInterface().dropTable("users", { cascade: true });
+  await sequelize
+    .getQueryInterface()
+    .dropTable("volunteers", { cascade: true });
+  await sequelize.getQueryInterface().dropTable("checkin", { cascade: true });
+  await sequelize.getQueryInterface().dropTable("content");
```

Comment:
for seeding down, i believe we should do `bulkDelete` instead of `dropTable`.

https://sequelize.org/v5/class/lib/query-interface.js~QueryInterface.html#instance-method-bulkDelete. I think we will need to have both truncate and cascade options as true in order for it to work

Reply:
seems like that's for version 5? Not seeing it in query options for v2